### ISSUE TITLE
UiConfirm show Modal with cancel CSS issue

### ISSUE THREAD

**Issue by fuchao2012** [JosephusPaye/Keen-UI#205]
When first pop up modal window, just like this

![image](https://cloud.githubusercontent.com/assets/5637185/22576231/57928b90-e9f5-11e6-972a-af795521057a.png)

The cancel might have some CSS problem.

**Comment by EmilMoe** [comment]
Also in the demo cases in the doc's I noticed this error.

**Comment by JosephusPaye** [comment]
It's not a bug, that shows which button is focused (in this case the Cancel button).

**Comment by EmilMoe** [comment]
But this is not the way Google Material sets focus and it gives the impression that it's a bug. A very eye catching one for me and seemingly more.

Google simply darkens the background a bit to show highlight https://material.angularjs.org/latest/demo/button

**Comment by JosephusPaye** [comment]
While it does seem weird, this is actually in the specs: https://material.io/guidelines/components/buttons.html#buttons-style under **Behavior** section.

It is also a place that Angular Material deviates from the specs, see https://github.com/angular/material/issues/142.

I will look into it again to see how it can be tweaked visually to look less like the focus is "stuck".

**Comment by JosephusPaye** [comment]
![keen-ui-focus-ring-animation](https://user-images.githubusercontent.com/5924865/43458189-911d3ef0-950c-11e8-849d-697951949a33.gif)

This was changed in 941e0a9750547822c2cc74c9d47d2c02e2b741df, to be released soon. Now the focus ring animates to fill the button.

To try it:

```sh
# npm
npm install keen-ui@JosephusPaye/Keen-UI#next

# yarn
yarn add keen-ui@JosephusPaye/Keen-UI#next
```

See the [changelog](https://github.com/JosephusPaye/Keen-UI/blob/next/CHANGELOG.md) for all changes.

**Comment by JosephusPaye** [comment]
Released in [v1.1.0](https://github.com/JosephusPaye/Keen-UI/releases/tag/v1.1.0).
